playbook_json_ces.yml

```diff
@@ -7,19 +7,19 @@
   vars:
     - scale_install_localpkg_path: /root/Spectrum_Scale_Advanced-5.1.0.0-x86_64-Linux-install
   roles:
-     - core/precheck
-     - core/node
-     - core/cluster
-     - gui/precheck
-     - gui/node
-     - gui/cluster
-     - gui/postcheck
-     - zimon/precheck
-     - zimon/node
-     - zimon/cluster
-     - zimon/postcheck
-     - callhome/precheck
-     - callhome/cluster
+     #- core/precheck
```

Comment:
Same as above comment . before commit run , git diff to make sure your required changes is getting for merge.